### PR TITLE
Added confirmation dialog when skipping after entering text

### PR DESCRIPTION
**Pull request by MehrJonas**
Adds a dialog asking the user to confirm whether they want to skip a question after they have already entered text to prevent accidentally losing an answer by clicking the wrong button as requested in https://github.com/LAION-AI/Open-Assistant/issues/2778

### CODE REVIEW

File: website/public/locales/en/common.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
   "sign_in": "Sign In",
   "sign_out": "Sign Out",
   "skip": "Skip",
+  "skip_confirmation": "Are you sure you want to skip?",
```
Comment (notmd) [inline website/public/locales/en/common.json:51]: Can you move this to task.json file?

File: website/src/components/Buttons/Skip.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  confirmSkip: boolean;
+}
+
+export const SkipButton = ({ ...props }: SkipButtonProps) => {
+  const { t } = useTranslation();
+  const { isOpen, onOpen: showModal, onClose: closeModal } = useDisclosure();
+
+  const onClick = () => {
+    if (props.confirmSkip) {
+      showModal();
```
Comment (notmd) [inline website/src/components/Buttons/Skip.tsx:26]: Also can you only show the modal when the input is not empty?

Comment (MehrJonas) [reply website/src/components/Buttons/Skip.tsx:26]: I'm already doing that... that's what the confirmSkip flag is for. I'm setting that to true if taskstatus.replyValidity === "VALID" (see TaskControls.tsx), which corresponds to a non-empty reply field.

Comment (notmd) [reply website/src/components/Buttons/Skip.tsx:26]: ah thanks

**Review by notmd — APPROVED** [review]

**Pull request merged**